docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -0,0 +1,137 @@
+# Use Kaito in AKS Arc
+This article describes how to deploy AI models on AKS arc with AI toolchain operator (KAITO). The AI toolchain operator (KAITO) is a managed add-on for all AKS, and it simplifies the experience of running OSS AI models on your AKS clusters. You may follow the workflow below to enable this feature:
+1.	Create a Node pool with GPU
+2.	Deploy KAITO operator
+3.	Deploy AI model
+4.	Validate the deployment
+
+## Supported Models
+Currently KAITO supports models such as Falcon, Phi2, Phi3, Llama2, Llama2Chat, Mistral. Please refer to KAITO’s [readme](https://github.com/Azure/kaito/blob/main/presets/README.md) file for the latest models. 
+
+## Prerequisite
+1.	Before you begin, please make sure you have the following details from your infrastructure administrator:
+    - An AKS cluster that's up and running.
+    - We recommend using Linux machine for this feature.
+    - Your local kubectl environment configured to point to your AKS cluster.
+        - Run `az aksarc get-credentials --resource-group <ResourceGroupName> --name <ClusterName>  --admin` to download the kubeconfig file.
+2.	Make sure your HCI cluster is enabled with GPU, you can ask your infrastructure administrator to set it up for you. You also need to identify the right VM SKUs for your AKS cluster before creating the node pool. The instruction can be found at [use GPU for compute-intensive workloads](https://learn.microsoft.com/en-us/azure/aks/hybrid/deploy-gpu-node-pool).
+3.	Make sure the helm and kubectl are installed in your local machine.
+    - If you need to install or upgrade, please see instruction from [Install Helm](https://helm.sh/docs/intro/install/).
+    - If you need to install kubectl, please see instructions from [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+## Create a GPU Node Pool
+<details>
+<summary><b>Using Azure Portal</b></summary>
+<div align="middle">
+  <img src="img/aksarc_nodepool_creation_portal.png" width=80% title="create nodepool from azure portal" alt="create nodepool from azure portal">
+</div>
+</details>
+
+Run following Az command to provision node pool, available GPU sku can be found [here](https://learn.microsoft.com/en-us/azure/aks/hybrid/deploy-gpu-node-pool#supported-vm-sizes)
+
+```bash
+az aksarc nodepool add --name "samplenodepool" --cluster-name "samplecluster" --resource-group "sample-rg" –node-vm-size "samplenodepoolsize" –os-type "Linux"

```

Comment:
Please make sure the nodepool name here is consistent with node name listed below.,